### DISCUSSION TITLE
List of elegible schools in the country?

### DISCUSSION THREAD

**Discussion by johiny** [community/community#49817] [GitHub Education] [answered]
Hi, I was exploring GitHub education and reading its FAQ, but I don't see a list or something about which schools have partnership with GitHub, I read in the last days that Mexico and Colombia enter the list of available countries, but how can I know which school are available in Colombia for example?
thx.

**Comment by enviGit** [comment, marked as answer]
### Hi Johiny!

GitHub Education is available to all accredited universities and colleges around the world, regardless of whether they have a specific partnership with GitHub or not. This means that any school that is officially recognized as an accredited higher education institution can take advantage of the benefits that GitHub Education offers, such as free access to GitHub Pro and other resources.

To find out if your school is eligible for GitHub Education benefits, you can visit the GitHub Education website and fill out the application form. During the application process, you will be asked to provide information about your school, such as its name, location, and accreditation status.

If your school is not currently listed on the GitHub Education website, it does not necessarily mean that it is ineligible for benefits. You can still apply and provide the necessary information to determine your eligibility.

In terms of specific schools that are eligible in a particular country, there is no comprehensive list available publicly. However, you can reach out to your school's IT department or academic office to inquire about their participation in GitHub Education or other similar programs. Additionally, you can contact GitHub support directly for more information or assistance with the application process.

**Reply by blancotm** [reply]
Alo soy Fernando Blanco de 51 años me operaron de del colon ace 6 o 7 mese no puedo trajara pues tengo mu cho tiempo libre y me dista el código y el 3d asi me aserque aki igno ro todo lo ilegal y legal de podi esto
Mi trabajo es modificar autos off road 
Ase un mes usando blender experimentando empeso a fallas lo reintale el caso que antier enpedo de nuevo y em pese aver que Windows emoedo a blokear mis dir y carpetas pense taner un virus que sun no lo descarto asi todo y no se ni lo que ise

**Reply by samus-aran** [reply]
Thank you @enviGit for your detailed answer. The only section I would correct you on is that we don't direct users to support if they are requesting for **individual application reviews** or **application statuses**. We recently made an[ announcement on the support available for GitHub Education](https://github.com/orgs/community/discussions/49366) 👍

**Comment by Ntshangase** [comment]
The best way is to just **TRY**, Chances are you will find the school if it's registered in that country.

**Comment by NanduBit** [comment]
I have recently made a list of school which are directly eligible for student developer benefits in the GitHub Student Program... 
I have curated over 22000+ schools and their verified domains with other details about them. You can check out the [School List Here](https://github.com/NanduWasTaken/GitHub-Student-Program-School-List). This doesn't contain the every school who are eligible but contains a decent chunk of it...(Eg: MIT is not in this list but is eligible in the program).. i couldn't collect every single school since it is a very hectic task and I would appreciate if someone is to help me wth it 🙏 

https://github.com/NanduWasTaken/GitHub-Student-Program-School-List
